src/tools/s2wasm.cpp

```diff
@@ -175,21 +175,21 @@ int main(int argc, const char *argv[]) {
     linker.linkArchive(lib);
   }
 
-  if (generateEmscriptenGlue) {
-    emscripten::generateRuntimeFunctions(linker.getOutput());
-  }
-
   linker.layout();
 
-  std::stringstream meta;
+  std::string metadata;
   if (generateEmscriptenGlue) {
-    if (options.debug) std::cerr << "Emscripten gluing..." << std::endl;
-    if (allowMemoryGrowth) {
-      emscripten::generateMemoryGrowthFunction(linker.getOutput().wasm);
+    Module& wasm = linker.getOutput().wasm;
+    if (options.debug) {
+      std::cerr << "Emscripten gluing..." << std::endl;
+      WasmPrinter::printModule(&wasm, std::cerr);
```

Comment:
we probably don't want to print the whole module here; I assume this is leftover from you debugging this PR?

Reply:
This is from the old `linker.emscriptenGlue`, `if (debug) WasmPrinter::printModule`

Morally I'm ok with printing the pre-glue module if you're debugging s2wasm.